governance/SDK.md

```diff
@@ -0,0 +1,163 @@
+
+NOTE: this process is valid only for security patches, not for bugs or
+enhancements.
+
+### Handover to a new maintainer/group of maintainers
+
+If a project is not meeting the criteria 1, 2, 4, 5, 6, 7, 8, the community
+MAY decides to handover the project to a new maintainer/group of
+maintainers. The community can perform the handover to a new maintainer
+if all the following conditions are met:
+
+* A vote to add the new maintainer MUST have **already taken place** with
+  all people entitled to vote abstaining.
+* One of the following conditions had been met:
+  * The new maintainer(s) already meet the requirements enlisted in [New maintainers section](#new-maintainers)
+  * The new maintainer(s) performed the required PRs, Reviews and Code changes to meet the [New maintainers criteria](#new-maintainers) but they were unable to merge them due to inactivity of the current maintainer(s).
+
+The new maintainer must show the good will to keep the project healthy and
+community could ask for an eventual plan to improve the quality of the project.
+
+Once a new maintainer is identified, the community can proceed with a vote
+to handover the project using the
+[Asynchronous voting process](#asynchronous-voting-process). If more than one
+new maintainer wants to handover the project, separate voting process must be
+done for all of them, and both of them must follow the same voting criteria (and
+not the ones from [New maintainers section](#new-maintainers)).
+
+The voting criteria are:
+
+- 1 week to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+### Archive a project
+
+If a project is not following the criteria 1, 2, 4, 5, 6, 7, 8, the community MAY decides to archive the project. Prior to archiving, the community SHOULD first consider performing a search for a new maintainer to [handover the project](#handover-to-a-new-maintainergroup-of-maintainers).
+
+If no new maintainer is found, the community can proceed to archive the project
+using the [Asynchronous voting process](#asynchronous-voting-process).
+
+The voting criteria are:
+
+- 2 weeks to vote
+- 2/3 of the agreement to accept the vote
+- All _sdk maintainers_ are entitled to vote
+
+## Asynchronous voting process
+
+In order to ensure the democratic management of debates in the community, we
+define an asynchronous voting process using GitHub issues.
+
+The voting is defined by:
+
+- The proposed community change to vote
+- The set of people/groups entitled to vote
+- A voting period
+- The percentage of agreements required to accept the change
+- A voting manager, a member of the community entitled to manage the vote. The voting manager MUST be part of people entitled to vote.
+
+To start a vote, the voting manager opens an issue in the
+[CloudEvents Spec](https://github.com/cloudevents/spec/issues) with:
+
+- A binary question to be voted on
+- Tags of the entitled people/groups to vote (depending on the question)
```

Comment:
Identifies the list of people who are entitled to vote

Reply:
rephrased a bit